frontend/src/hooks/useGetBoardInfo.ts

```diff
@@ -120,6 +120,17 @@ export const useGetBoardInfoEffect = (): useGetBoardInfoInterface => {
           });
       });
 
+      dispatch(
+        updateMetricsPageFailedTimeRangeInfos(
+          dateRangeCopy.map((dateRange) => ({
+            startDate: formatDateToTimestampString(dateRange.startDate!),
+            errors: {
+              isBoardInfoError: undefined,
```

Comment:
`isBoardInfoError` is a boolean, why assign it an `undefined` ....
same as the below parts

Reply:
isBoardInfoError  is boolean or no value. undefined means there is no error status, that is the data is still loading. Here we set these errors to undefined to clear the error status.